packages/core/src/llm/model/gateways/constants.ts

```diff
@@ -0,0 +1 @@
+export const SUPPORTED_PROVIDERS = ['anthropic', 'google', 'openai', 'openrouter', 'xai'];
```

Comment:
Does this mean that these are the only supported providers?

Reply:
If that's the case we should remove docs for all of the other providers

Reply:
https://github.com/mastra-ai/mastra/pull/8559/commits/e1ff8fc0e8840dc745e145e080c9f2e48a6c2a89